<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>busca</class>
 <widget class="QDialog" name="busca">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Busca no banco</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QTextEdit" name="enunciado_questao">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="resposta_questao"/>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Busca</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2" colspan="2">
       <widget class="QComboBox" name="materia">
        <property name="currentText">
         <string/>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAlphabetically</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QComboBox" name="assunto"/>
         </item>
         <item>
          <widget class="QListWidget" name="list_questoes"/>
         </item>
         <item>
          <widget class="QPushButton" name="buscar_bd">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="voltar">
        <property name="text">
         <string>Voltar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="adicionar_question">
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
